src/Command/InfectionCommand.php

```diff
@@ -435,8 +445,10 @@ private function getCodeCoverageData(string $testFrameworkKey): LineCodeCoverage
 
     private function getTestFrameworkExtraOptions(string $testFrameworkKey): TestFrameworkExtraOptions
     {
-        $extraOptions = $this->input->getOption('test-framework-options')
-            ?? $this->container['infection.config']->getTestFrameworkOptions();
+        /** @var Configuration $config */
+        $config = $this->container[Configuration::class];
```

Comment:
Pretty much everywhere we are using the values from the config directly, no longer from the input (when the values _can_ come from the config)